docs/get-started/training-pytorch.md

```diff
@@ -1,33 +1,38 @@
 ---
-title: ORT Training with PyTorch
```

Comment:
We need both large model training and on-device training.

Reply:
I just changed the name from `ORT Training with PyTorch` to `Large Model Training`.

I have added a new page for `On-Device Training`.

I did not remove this page. Maybe I missed your point.